collect_app/src/main/java/org/odk/collect/android/javarosawrapper/FormController.java

```diff
@@ -371,11 +372,11 @@ private FormInstance getInstance() {
     private boolean groupIsFieldList(FormIndex index) {
         // if this isn't a group, return right away
         IFormElement element = formEntryController.getModel().getForm().getChild(index);
-        if (!(element instanceof GroupDef)) {
+        if (!(element instanceof GroupDef) || element.getAppearanceAttr() == null) {
             return false;
         }
 
-        return ODKView.FIELD_LIST.equalsIgnoreCase(element.getAppearanceAttr());
+        return element.getAppearanceAttr().toLowerCase(Locale.ENGLISH).contains(ODKView.FIELD_LIST);
```

Comment:
It might be a good opportunity to move this const to `WidgetAppearanceUtils` where we store all appearances? That name `WidgetAppearanceUtils` might require refactoring.

Reply:
Thanks! I did so and renamed to `Appearances` following the Java pattern from `Files`, `Collections`, etc. We're not very consistent with naming utilities and I also considered `AppearanceUtils`. I prefer `Appearances` because I think it's really nice when accessing the constants and pretty good for method calls, too.